core/lib/eth_client/src/lib.rs

```diff
@@ -65,6 +65,13 @@ impl Options {
     }
 }
 
+/// Information about the base fees provided by the L1 client.
+#[derive(Debug, Clone, PartialEq, Eq, Default)]
+pub struct BaseFees {
+    pub base_fee_per_gas: u64,
+    pub base_fee_per_blob_gas: U256,
```

Comment:
Bikeshedding: Discrepancy between types of these fields looks weird; it could make sense to define both as `U256`.

Reply:
My reasoning here was that `BaseFees` is an internal interface, and internally we only treat `base_fee_per_gas` as `u64`, and `base_fee_per_blob_gas` as `U256`.
AFAIR treating gas as `u64` is considered a problem (it has issues, for example, with custom base tokens), but fixing it is not scheduled just yet -- so I decided to be consistent here and not introduce more type variance across components.